packages/types/src/gif.ts

```diff
@@ -71,7 +71,7 @@ export default interface IGif {
     source_tld: string
     source_post_url: string
     is_indexable: boolean
-    is_sticker: boolean
```

Comment:
@kylebcarlson maybe remove this `is_sticker` change and let's release just the `primary_site` change for now

Reply:
sounds good!

Reply:
@kylebcarlson can you add in `is_sticker` back? sorry I meant remove the change, not remove the property :)